cms/djangoapps/contentstore/views/course.py

```diff
@@ -458,11 +458,19 @@ def filter_ccx(course_access):
     return courses_list.values(), in_process_course_actions
 
 
-def _accessible_libraries_iter(user):
+def _accessible_libraries_iter(user, org=None):
     """
-    List all libraries available to the logged in user by iterating through all libraries
+    List all libraries available to the logged in user by iterating through all libraries.
```

Comment:
If you are going to do this, I would recommend also removing the checkbox about showing libraries. With the smaller number of libraries shown, I think you might as well always show the ones for the given org.